cli/tools/init/templates/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+  "deno.lint": true,
+  "deno.unstable": true
```

Comment:
Shouldn't this only be?

```json
{
  "deno.enable": true
}
```

Then let the other settings be whatever the defaults are?